stories/RefreshCache.stories.js

```diff
@@ -128,3 +129,74 @@ class AppWithRefresh extends Component {
 }
 
 stories.add('with a refresh button', () => <AppWithRefresh />);
+
+class StopWatch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { count: 0 };
+  }
+
+  componentWillReceiveProps = nextProps => {
+    if (nextProps.refresh) {
+      this.setState({ count: 0 });
+    }
+  };
+
+  componentDidMount = () => {
+    setInterval(
+      () =>
+        this.setState(prevState => ({
+          count: prevState.count + 1,
+        })),
+      1000
+    );
+  };
+
+  render = () => <span>{this.state.count}s elasped since refresh</span>;
+}
+
+StopWatch.propTypes = {
+  refresh: PropTypes.bool,
+};
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      refresh: false,
+    };
+  }
+
+  componentDidMount = () => {
```

Comment:
Don't need to use a class property, the regular method should work `componentDidMount() { ... }`